### DISCUSSION TITLE
Help installing in Ubuntu 20.04

### DISCUSSION THREAD

**Discussion by lee-costa** [ghostfolio/ghostfolio#965] [Q&A]
Hello,

I am looking to try ghostfolio but have very little knowledge with Docker. I have a fresh Ubuntu 20.04 install and when trying the commands in the installation guide I seem to get stuck on this part:

    ~/ghostly$ sudo docker-compose -f docker-compose.yml exec ghostly_ghostfolio_1 yarn database:setup
    ERROR: No such service: ghostly_ghostfolio_1

I am sure I am doing something wrong so I will restore my snapshot and try again but was wondering if someone could provide some simple installations instructions.

I will have a fresh docker (and docker compose) installed and ready to go.

Thank you.

**Comment by lee-costa** [comment]
Well, I think I made some progress. I am now at this step but this happens:

        ~/ghostfolio$ sudo docker-compose -f docker/docker-compose.yml exec ghostfolio yarn database:setup
        yarn run v1.22.19
        $ yarn database:push && yarn database:seed
        $ prisma db push
        Prisma schema loaded from prisma/schema.prisma
        Datasource "db": PostgreSQL database "ghostfolio-db", schema "public" at "postgres:5432"

        Error: P1000: Authentication failed against database server at `postgres`, the provided database credentials for `user` are not valid.

        Please make sure to provide valid database credentials for the database server at `postgres`.
        error Command failed with exit code 1.
        info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
        error Command failed with exit code 1.
        info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.


my .env file:

        COMPOSE_PROJECT_NAME=ghostfolio-development

        # CACHE
        REDIS_HOST=localhost
        REDIS_PORT=6379

        # POSTGRES
        POSTGRES_DB=ghostfolio-db
        POSTGRES_USER=user
        POSTGRES_PASSWORD=password

        ACCESS_TOKEN_SALT=GHOSTFOLIO
        ALPHA_VANTAGE_API_KEY=
        DATABASE_URL=postgresql://${POSTGRES_USER}:${POSTGRES_PASSWORD}@localhost:5432/${POSTGRES_DB}?sslmode=prefer
        JWT_SECRET_KEY=123456
        PORT=3333

Do I need to setup a user/password under postgres?

**Reply by jte0711** [reply]
hey, do you find an answer to this problem?
